modules/sc-mesh-secure-deployment/src/nats/initd/S90APoint

```diff
@@ -53,21 +53,19 @@ case "$1" in
        fi
         ;;
 
-
     stop)
         if "$daemon" --running --name "$name" --pidfiles "$pidfiles"
         then
             echo "Stopping $name..."
             "$daemon" --stop --name "$name" --pidfiles "$pidfiles"
             echo done.
 
-            NEW_PID=$(ps ax | grep -E "[h]ostapd-$RADIO_INDEX" | awk {'print $1'})
+            NEW_PID=$(ps ax | grep -E "[h]ostapd-$RADIO_INDEX" | awk "{'print $1'}")
```

Comment:
The `awk` command will fail, replace it to: `awk '{print $1}'`

Or even join it with the `grep`: `ps ax | awk "/ostapd-$RADIO_INDEX/{print \$1}"`

:bagel:

Reply:
Yes, this PR is draft and under testing currently and not including the final changes.